specification/containerservice/Fleet.Management/clusterresourceplacement.tsp

```diff
@@ -0,0 +1,193 @@
+model PlacementPolicy {
+  @doc("Type of placement. Default is PickAll.")
+  placementType?: PlacementType;
+
+  @doc("ClusterNames contains a list of names of MemberCluster to place the selected resources. Only valid if the placement type is PickFixed.")
+  clusterNames?: string[];
+
+  @doc("Affinity contains cluster affinity scheduling rules. Defines which member clusters to place the selected resources. Only valid if the placement type is PickAll.") // don't forget to update this when we add PickN.
+  affinity?: Affinity;
+
+  @doc("If specified, the ClusterClusterResourcePlacement's Tolerations. Tolerations cannot be updated or deleted.")
+  @OpenAPI.extension("x-ms-identifiers", [])
+  tolerations?: Toleration[];
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("PlacementType identifies the type of placement.")
+union PlacementType {
+  @doc("PickAll selects all clusters that match the specified requirements.")
+  PickAll: "PickAll",
+
+  @doc("PickFixed selects a fixed set of clusters")
+  PickFixed: "PickFixed",
+
+  string,
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("Affinity is a group of cluster affinity scheduling rules.")
+model Affinity {
+  @doc("ClusterAffinity contains cluster affinity scheduling rules for the selected resources.")
+  clusterAffinity?: ClusterAffinity;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("ClusterAffinity contains cluster affinity scheduling rules for the selected resources.")
+model ClusterAffinity {
+  @doc("If any affinity requirements specified by this field are not met at scheduling time, the resource will not be scheduled onto the cluster.")
+  requiredDuringSchedulingIgnoredDuringExecution?: ClusterSelector;
+}
+@added(Versions.v2025_03_02_naar)
+@doc("ClusterSelector allows specifying cluster selector terms.")
+model ClusterSelector {
+  @doc("ClusterSelectorTerms is a list of cluster selector terms. The terms are `ORed`.")
+  @OpenAPI.extension("x-ms-identifiers", [])
+  clusterSelectorTerms: ClusterSelectorTerm[];
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("ClusterSelectorTerm allows specifying what clusters in the fleet are in scope. If you specify both label and property selectors in the same term, the results are AND'd.")
+model ClusterSelectorTerm {
+  @doc("LabelSelector is a label query over all the joined member clusters. Clusters matching the query are selected.")
+  labelSelector?: LabelSelector;
+
+  @doc("PropertySelector is a property query over all joined member clusters. Clusters matching the query are selected.")
+  propertySelector?: PropertySelector;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("A label selector is a label query over a set of resources. The result of matchLabels and matchExpressions are ANDed. An empty label selector matches all objects. A null label selector matches no objects.")
+model LabelSelector {
```

Comment:
this comes from metav1 - is there really no way this can be reflected in our tsp namespacing?